### DISCUSSION TITLE
using (MagickImage image = new MagickImage(nefPath)) error: Unsupported file format or not RAW file `xxx.NEF' @ error/dng.c/ReadDNGImage/574

### DISCUSSION THREAD

**Discussion by lt090004114** [dlemstra/Magick.NET#1504] [Help]
using (MagickImage image = new MagickImage(nefPath)) 
 error: Unsupported file format or not RAW file `xxx.NEF' @ error/dng.c/ReadDNGImage/574
xxx.NEF info: Nikon Z 9
![Uploading error_1.png…]()

**Comment by dlemstra** [comment]
It looks like you got an unsupported NEF file. It might be possible that support for that will be added in `libraw` in the future but you will not be able to read that file with this library right now.

**Reply by lt090004114** [reply]
Thanks.
I wish that will be added in libraw in the future earlier.

**Reply by lt090004114** [reply]
If it's convenient, I can ask you when will you add support for NEF files to the library?

**Comment by dlemstra** [comment]
NEF files are supported. But not all of them are supported. And I don't maintain the libraw library where support needs to be added so I cannot answer your question.
